locales/fr.yml

```diff
@@ -75,18 +76,18 @@ fr:
     ban_user: "Bannir l'utilisateur ?"
     banned: Banni
     banned_user: "Utilisateur banni"
-    cancel: Signalé
-    dont_like_username: "Dislike username"
+    cancel: Annuler
+    dont_like_username: "Ne pas aimer le nom d’utilisateur"
     flaggedaccounts: "Noms d'utilisateurs signalés"
     flags: Signalements
-    impersonating: Impersonation"
+    impersonating: "Impersonation"
```

Comment:
Not your translation, but this is a mistranslation, I would use "Usurpation d’identité"

Reply:
I thought about changing that too.

Reply:
Fixed